samples/src/main/java/common/PaymentClient.java

```diff
@@ -72,14 +72,15 @@ private Metadata getMetadata() {
         return metadata;
     }
 
-    public CompletableFuture<PayResponse> sendPayment(String terminalId, BigDecimal amount, java.util.function.Consumer<String> onPending) {
+    public CompletableFuture<PayResponse> sendPayment(boolean showTips, String terminalId, BigDecimal amount, java.util.function.Consumer<String> onPending) {
```

Comment:
Cosmetics: I would put `showTips` after `amount`, so they are in logical order:
- which terminal do we use?
- how much do we want to charge on that terminal?
- do we want to add tips on top of that amount?

Reply:
Also, as `showTips` is optional in the proto, we should have a variant os the `sendPayment` function without `showTips`.

Reply:
In order to resolve this comment, I think I need more clarity on the use case.

As I understand the use case for terminal payments, we will do the configuration either to have tips or not to have tips. 
In order to replicate the use case , when selection is made to execute the terminal payment, I want to ask the developer which use case he wants to run 
i.e. whether to send the payment with tips or without tips. In both the scenario we will have the value for showTips. so I think method with show tips parameter is enough.

Or As this is a sample code we want to let developer know that you can create method without show tips as well?

Reply:
This is just the sample code, I wouldn't worry about making too much more complexity than what it already is. Showing all the possible options is more than fine, even setting the value to true since we want to demonstrate it. We don't need the sample to replicate the "default" behaviour, that the developer can find on the auto-completion code.